addon/components/au-control-radio.js

```diff
@@ -12,7 +12,7 @@ export default class AuControlRadio extends Component {
   @action
   onChange(event) {
     if (this.args.onChange) {
-      this.args.onChange(event.target.value);
+      this.args.onChange(event.target.checked, event.target.value, event);
```

Comment:
by changing the order of the parameters here, you've introduced a breaking change.

Reply:
Yes, that did come to mind when I coded this. I asked Dieter if the radio button was used somewhere critical and other then the contact form nothing was else was mentioned, I can change the order around so value gets passed in first :)